src/docs/development/data-and-backend/state-mgmt/options.md

```diff
@@ -62,6 +62,7 @@ A state container approach familiar to many web developers.
 * [Animation Management with Redux and Flutter][],
   a video from DartConf 2018 [Accompanying article on Medium][]
 * [Flutter Redux package][]
+* [Redux Saga Middleware Dart and Flutter][]
```

Comment:
Can you please add "by Bilal Uslu" at the end of line like the following;

* [Redux Saga Middleware Dart and Flutter][], by Bilal Uslu

Reply:
I have added the author name to the new commit in this PR